dist/build.js

```diff
@@ -1626,6 +1629,15 @@ utils.getBooleanOrNull = function(a) {
 utils.delay = function(a, b) {
   isNaN(b) || 0 >= b ? a() : setTimeout(a, b);
 };
+utils.getClientHints = function() {
+  navigator.userAgentData ? navigator.userAgentData.getHighEntropyValues(["model", "platformVersion"]).then(function(a) {
```

Comment:
does this api expose brand on mobile devices? can we send that too?

Reply:
@mack-branch  brand is a low entropy value which can be directly accessed using
`console.log(navigator.userAgentData.brands);`
which results in an array something like 

```
[
    {
        "brand": "Chromium",
        "version": "110"
    },
    {
        "brand": "Not A(Brand",
        "version": "24"
    },
    {
        "brand": "Google Chrome",
        "version": "110"
    }
]
```
To check whether the device type is a mobile or not. we can use `console.log(navigator.userAgentData.mobile);` :  true if this is a mobile device.